src/System.IO/src/netstandard15aot/project.json

```diff
@@ -0,0 +1,19 @@
+{
+  "frameworks": {
+    "netstandard1.5": {
```

Comment:
Don't do that.  Appending aot to the framework moniker is not valid.  'netstandard13aot' is just an arbitrary string the build tooling is using to enable cross-compilation.